ui-tests/cypress/integration/components/ux_fragments.spec.js

```diff
@@ -1,141 +1,783 @@
+        openFragmentsPage()
+            .then(page => {
+              page.getContent().getWidgetFilter().select('Content');
+              page.getContent().getPluginFilter().select('jacms');
+              page.getContent().clickSearchSubmitButton();
+            })
+            .then(page =>
+                page.getContent().getTableRows().should('have.length', 3)
+                    .each(row => cy.get(row).children(htmlElements.td).eq(2).should('contain', 'Content'))
+                    .each(row => cy.get(row).children(htmlElements.td).eq(4).should('contain', 'jacms')));
+        cy.get('@currentPage')
+          .then(page => {
+            page.getContent().getPagination().getItemsCurrent().invoke('text').should('be.equal', '1-3');
+            page.getContent().getPagination().getItemsTotal().invoke('text').should('be.equal', '3');
+          });
+      });
+      it([Tag.FEATURE, 'ENG-3522'], 'Search with fragment code and filters', () => {
+        openFragmentsPage()
+            .then(page =>
+                page.getContent().getSearchCodeInput().then(input => page.getContent().type(input, 'row')))
+            .then(page => {
+              page.getContent().getWidgetFilter().select('Content List');
+              page.getContent().getPluginFilter().select('jacms');
+              page.getContent().clickSearchSubmitButton();
+            })
+            .then(page =>
+                page.getContent().getTableRows().should('have.length', 1)
+                    .each(row => cy.get(row).children(htmlElements.td).eq(0).should('contain', 'row'))
+                    .each(row => cy.get(row).children(htmlElements.td).eq(2).should('contain', 'Content List'))
+                    .each(row => cy.get(row).children(htmlElements.td).eq(4).should('contain', 'jacms')));
+        cy.get('@currentPage')
+          .then(page => {
+            page.getContent().getPagination().getItemsCurrent().invoke('text').should('be.equal', '1-1');
+            page.getContent().getPagination().getItemsTotal().invoke('text').should('be.equal', '1');
+          });
+      });
+      it([Tag.FEATURE, 'ENG-3522'], 'No Results from a non-existing fragment', () => {
+        openFragmentsPage()
+            .then(page =>
+                page.getContent().getSearchCodeInput().then(input => page.getContent().type(input, fragment.code.substring(3, 7))))
+            .then(page => page.getContent().clickSearchSubmitButton())
+            .then(page =>
+                page.getContent().getTableRows().should('have.length', 0));
+        cy.get('@currentPage')
+          .then(page => {
+            page.getContent().getPagination().getItemsCurrent().invoke('text').should('be.equal', '0-0');
+            page.getContent().getPagination().getItemsTotal().invoke('text').should('be.equal', '0');
+          });
+      });
 
-  describe(['ENG-2680'], 'Fragments Browsing', () => {
+      //BUG: after search doesn't show current number of fragments
```

Comment:
move this to a PR comment, do not leave it in the code or we will have to do a new PR when the bug is fixed